### ISSUE TITLE
Due to some reasons media queries are not effective!

### ISSUE THREAD

**Issue by iaminamcom** [bep/hugo-starter-tailwind-basic#7]
Hi
I'm observing that due to some reasons things messed up. I don't know if it's in only my case or you guys configured it by choice.
Check the screenshot here:
![image](https://user-images.githubusercontent.com/34743206/107045316-3210cc00-67e7-11eb-8105-2c2d429d4bd5.png)
See how even in mobile preview class with in media query `(min-width: 768px)` is overriding the default class. I'm confused!

I was excited to use this starter template by Bep and other but currently having bad time. BTW, I'm not new to tailwindcss.

**Comment by bep** [comment]
> I don't know if it's in only my case or you guys configured it by choice.

The provided configuration is rather small; have you checked?

**Comment by iaminamcom** [comment]
Thanks for the response! I went through whole configurations. I found no evident reason for it. Is this working good in your case?

**Comment by divinerites** [comment]
They changed things for breakpoints in v2. May be related ?

See https://tailwindcss.com/docs/upgrading-to-v2#configure-your-breakpoints-explicitly

**Comment by iaminamcom** [comment]
I don't think so. I migrated my content to https://github.com/dirkolbrich/hugo-theme-tailwindcss-starter and it's quite perfect
